Doesn't work on Raspberry Pi 4

I got this message when trying to build the addic7ed project that is using your plugin.

Also used on MacOS, and working well.

```
pi@raspberrypi:~/go/src/github.com/addic7ed $ go build
# github.com/masatana/go-textdistance
../../../pkg/mod/github.com/masatana/go-textdistance@v0.0.0-20171214133840-59954f2a9d67/util.go:7:12: constant 9223372036854775807 overflows int
```

--- a/util.go
+++ b/util.go
@@ -4,7 +4,7 @@ import "math"
 
 // Min returns the minimum number of passed int slices.
 func Min(is ...int) int {
-	min := int(math.MaxInt64)
+	min := int64(math.MaxInt64)
 	for _, v := range is {
 		if min > v {
 			min = v

--- a/util.go
+++ b/util.go
@@ -1,12 +1,10 @@
 package textdistance
 
-import "math"
-
 // Min returns the minimum number of passed int slices.
 func Min(is ...int) int {
-	min := int64(math.MaxInt64)
-	for _, v := range is {
-		if min > v {
+	var min int
+	for i, v := range is {
+		if i == 0 || v < min {
 			min = v
 		}
 	}

